app/views/customers/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Customers</h1>
+
+<table>
+  <thead>
+    <tr>
+      <%# <th>First name</th> %>
+      <%# <th>Last name</th>  %>
+      <th>Full name</th>
+      <th>Phone</th>
+      <th>Email</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @customers.each do |customer| %>
+      <tr>
+        <%# <td><%= customer.first_name %> <%#</td> %>
```

Comment:
Lines 20-22 are still visible in HTML, you should also comment it out with "<! --    --> "